tests/v2/integration/cli/test_cli.py

```diff
@@ -37,7 +38,7 @@ def test_otx_cli(task: str, tmp_dir_path: Path) -> None:
         "--data.val_data_roots", TASK_CONFIGURATION[task]["val_data_roots"],
         "--max_epochs", "1",
         "--seed", "1234",
-        "--deterministic", "True",
+        "--deterministic", deterministic,
```

Comment:
I think we can set deterministic False for all tasks for a while

Reply:
quick question: isn't setting deterministic to True more safer than False?

Reply:
For perspective of test, it is reasonable to set this True since test should produce same output without code changes.
However sometime deterministic raise numerical error. I met this situation in my task. I guess Eugene also met some numerical errors during this PR. I think we should set this value as True eventually, but it is okay to leave as it is in this phase.

Reply:
I don't think it's possible at the moment. https://github.com/pytorch/pytorch/issues/107999 

`RuntimeError: upsample_bilinear2d_backward_out_cuda does not have a deterministic implementation, but you set \'torch.use_deterministic_algorithms(True)\'. You can turn off determinism just for this operation, or you can use the \'warn_only=True\' option, if \nthat\'s acceptable for your application. You can also file an issue at https://github.com/pytorch/pytorch/issues to help us prioritize adding deterministic support for this operation.\n'))`

Reply:
Okay, then how about setting deterministic False for a while? Detection task also couldn't support deterministic

Reply:
@jaegukhyun lets' change it to False.